app/javascript/chat/content.jsx

```diff
@@ -74,7 +74,6 @@ export default class Content extends Component {
     return (
       // eslint-disable-next-line jsx-a11y/click-events-have-key-events
       <div
-        role="presentation"
```

Comment:
I removed this because there is already a `role="button"` but this is problematic. Interactive elements like a button cannot contain buttons. This fixes the lint issue, but this will need to be revisited. I've. added a comment in the code about this.